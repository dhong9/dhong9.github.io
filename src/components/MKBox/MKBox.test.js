/**
=========================================================
* Danyo-1.2
=========================================================

* Copyright 2024 Danyo (https://www.danyo.tech)

Coded by www.danyo.tech

=========================================================

* The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
*/

// React testing libraries
import { render } from "@testing-library/react";

// Material Kit 2 React themes
import { ThemeProvider } from "@mui/material/styles";
import theme from "assets/theme";

// Component to test
import MKBox from "components/MKBox";

describe("MKBox", () => {
  it("renders gradient white Box", () => {
    const { container } = render(
      <ThemeProvider theme={theme}>
        <MKBox variant="gradient" color="white" />
      </ThemeProvider>
    );

    expect(container).toMatchSnapshot();
  });
});
