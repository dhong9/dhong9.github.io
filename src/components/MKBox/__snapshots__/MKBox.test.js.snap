// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MKBox renders gradient white Box 1`] = `
<div>
  <div
    class="MuiBox-root css-do2pae"
  />
</div>
`;
